log-warper.cabal

```diff
@@ -22,31 +22,27 @@ library
   exposed-modules:     System.Wlog
                        System.Wlog.CanLog
                        System.Wlog.FileUtils
+                       System.Wlog.Formatter
+                       System.Wlog.Handler
+                       System.Wlog.Handler.Simple
+                       System.Wlog.Handler.Syslog
                        System.Wlog.Launcher
+                       System.Wlog.Logger
                        System.Wlog.LoggerConfig
                        System.Wlog.LoggerName
                        System.Wlog.LoggerNameBox
                        System.Wlog.Roller
                        System.Wlog.Severity
                        System.Wlog.Wrapper
 
-                       -- This part is taken from hslogger library directly
-                       System.Log
-                       System.Log.Handler
-                       System.Log.Formatter
-                       System.Log.Handler.Simple
-                       System.Log.Handler.Syslog
-                       System.Log.Logger
-
   other-modules:       System.Wlog.Color
-                       System.Wlog.Formatter
                        System.Wlog.MemoryQueue
 
   build-depends:       aeson                >= 0.11.2.1
                      , ansi-terminal        >= 0.6.2.3
                      , base                 >= 4.7 && < 5
                      , bytestring           >= 0.10.8.1
-                     , containers
+                     , containers           >= 0.5.7.0
```

Comment:
Maybe delete all version bounds? I put them in the beginning because I was stupid. Now I think that using `stack` it's more convenient to not use lower-bounds. And `stack` also can add correct bounds on uploading to `hackage`.